src/components/HomeBody.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react'
+import Overlay from './Overlay'
+import Quote from './Quote'
+import Icons from './Icons'
+
+class HomeBody extends Component {
+  render () {
+    return (
+      <div className="home-main">
+        <Overlay/>
+        <section className="quotes">
+          <div className="quotes-container">
+            <Quote quote="I found an apple thanks to Dragonboard" url="https://1.bp.blogspot.com/-vw-cIkAWfDI/TeDVgYZljYI/AAAAAAAAAAQ/TLY7IGiDC7M/s1600/gravity2.png" name="Isaac Newton"/>
```

Comment:
Over 80 characters.

Please reformat long Components to place attributes on new lines, indented beneath opening tag:
```js
<Component attributeOne="this is a really long attribute and will cause additional attributes to be off page"
  attributeTwo="valueTwo"
  attributeThree="valueThree" />
```